BACnet.csproj

```diff
@@ -17,24 +18,26 @@
     <PackageIcon>logo.png</PackageIcon>
     <PackageIconUrl>https://raw.githubusercontent.com/ela-compil/BACnet/master/logo.png</PackageIconUrl>
   </PropertyGroup>
-
+  
   <ItemGroup>
     <None Include="logo.png" Pack="true" Visible="false" PackagePath="" />
   </ItemGroup>
 
   <!-- Vendor Packages -->
   <ItemGroup>
     <PackageReference Include="Common.Logging" Version="3.4.1" />
-    <PackageReference Include="Common.Logging.Core" Version="3.4.1" />
+    <PackageReference Include="PacketDotNet" Version="1.4.0" />
+    <PackageReference Include="SharpPcap" Version="6.1.0" />

```

Comment:
You have added here additional versions of PacketDotNet and SharpPcap packages. These are in conflict with references in conditional item groups.

Reply:
Oops.  I noticed that earlier but was thinking that somehow got pulled in with updating the framework version.  Should be removed now.